dotcom-rendering/src/model/enhanceTags.ts

```diff
@@ -0,0 +1,24 @@
+import type { FETagType, TagType } from '../types/tag';
+
+export const enhanceTags = (tags: FETagType[]): TagType[] =>
+	tags.map((tag) => enhanceTag(tag));
```

Comment:
If you so desire, this may be even clearer:

```suggestion
	tags.map(enhanceTag);
```

Reply:
I've shied away from this pattern since reading [this blog post](https://jakearchibald.com/2021/function-callback-risks/). Agreed that since we own/control the function the issue talked about in the post is less likely to crop up, but in general I'm happy to use the slightly more verbose form in all cases. What do you think?

Reply:
That’s a good point, and I think that if we wanted to follow this pattern we [should enforce it with an ESLint rule](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/no-array-callback-reference.md). In this specific case, and as both pattern are used extensively – we have [~280 arrow functions](https://github.com/search?q=repo%3Aguardian%2Fdotcom-rendering+%2F%5C.map%5C%28%5C%28%2F&type=code) and [~30 direct callbacks](https://github.com/search?q=repo%3Aguardian%2Fdotcom-rendering+%2F%5C.map%5C%28%5Cw%2B%2F&type=code) – I’ll let you decide which one you prefer!